pkg/controller/pod_driver.go

```diff
@@ -495,6 +506,10 @@ func (p *PodDriver) podReadyHandler(ctx context.Context, pod *corev1.Pod) error
 		return nil
 	}
 
+	if util.ParseClientVersion(pod.Spec.Containers[0].Image).SupportFusePass() {
+		return nil
+	}
```

Comment:
If you return here, cannot be store dev minor

Reply:
fixed